Women-In-Foss/IIT.md

```diff
@@ -0,0 +1,36 @@
+# IIT - Informatics Institute of Technology
+ 
+ A set of highly skilled and enthusiastic individuals who come together to build a community of free and open source development under the guidance of Mozilla Sri Lanka. At Mozilla Campus Club of IIT, we create awareness of all the free resources in the technology field and pave the way to many global challenges while aiming to interact with others to share, learn, teach, protect and build upon platforms, to make things that matter. Our mission is to encourage evolving developers to embrace their passion for technology.
+
+
+## Members
+
+* Santhusha Mallawatantri (Club Captain) [@LinkedIn](https://www.linkedin.com/in/santhusha-mallawatantri/)
```

Comment:
@Santhusha-Mallawatantri are you the club lead of Women in FOSS as well?

Reply:
Yes for Mozilla FOSS and WIF I'm the lead